test/hotspot/gtest/nmt/test_vmatree.cpp

```diff
@@ -284,6 +284,68 @@ TEST_VM_F(NMTVMATreeTest, LowLevel) {
   }
 }
 
+TEST_VM_F(NMTVMATreeTest, SetTag) {
+  auto i = [](MemTag f) -> uint8_t { return (uint8_t)f; };
+
+  // The gc/cds case with only reserved data
+  {
+    VMATree::SummaryDiff diff;
+    Tree::RegionData rd(NCS::StackIndex(), mtNone);
+    VMATree tree;
+
+    VMATree::SummaryDiff result = tree.reserve_mapping(0, 500, rd);
```

Comment:
How does using '0' gel with your find_enclosing_range, where NULL for start or end is "did not find"?

Reply:
Sorry, I don't see what the relevance is for this line of code with the rest of the PR :-). Of course, `nullptr` is a pointer type so can't be passed in as a numerical argument, as opposed to `NULL`.

Reply:
Ah, my mistake. Range.end/start is null, so the TreapNode* itself is null, not the address the TreapNode* represents.